tensorboard/webapp/metrics/views/main_view/card_group_toolbar_component.scss

```diff
@@ -0,0 +1,49 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+@import 'tensorboard/webapp/theme/tb_theme';
+@import '../common';
+
+:host {
+  @include metrics-sub-view;
+
+  .group-toolbar {
+    @include tb-theme-foreground-prop(border-top, border, 1px solid);
+    // border-top on group-toolbar needs to be offset on sticky header for the
+    // flawless UI.
+    top: -1px;
+    display: flex;
+
+    &:hover {
+      cursor: pointer;
+    }
+  }
+}
+
+.card-group:first-of-type .group-toolbar {
```

Comment:
You might be able to write

```scss
.card-group:first-of-type & {
  ...
}
```
in line 27.

Reply:
done, thanks for the suggestion